tf-101/1-why-iac.md

```diff
@@ -25,51 +25,74 @@ redirect: https://developer.oracle.com/tutorials/tf-101/1-why-iac/
 
-One of the most common and popular IaC tools is [HashiCorp Terraform](https://www.terraform.io), with other tools (such as [Pulumi](https://www.pulumi.com)) also being available.  The end goal with these tools is to manage infrastructure programmatically.
+## What's Next
 
-Some like to use tools that are predominantly in the realm of configuration management (such as [Ansible](https://www.ansible.com), [Chef](https://www.chef.io)/[Cinc](https://cinc.sh), etc.  These are certainly options for managing infrastructure, but you may be better off with tools that focus on *infrastructure management* (and less about *configuration management*).  OCI is no exception in supporting many of these different platforms.
+By now, you should be a little more familiar with IaC and ready to get started with Terraform!
 
-In the rest of this series, we're going to target using Terraform to manage OCI infrastructure.  Why Terraform?  It's fairly mature at this point, widely adopted and has support for a wide variety of platforms.  Many cloud platforms (including OCI) and IT systems support using Terraform to manage resources.  There are plenty of examples, platforms and strong user support for it, making it an ideal tool.  Terraform is targeted at managing infrastructure, not so much the configuration management side of things.  And it's super powerful when combined with a traditional configuration management tool like Ansible.
+Until your next lesson, happy coding! Take a look at the [next lesson](2-experiencing-terraform) in the Terraform 101 series and go through a very quick experience in using Terraform. From there, you'll dive into several aspects of how Terraform works.
 
-This series will take you through how to harness the power of infrastructure-as-code (IaC) in your environment.  If this is your first time using Terraform, you're a bit rusty, or you're just looking to fill any potential gaps in your knowledge, this should be a worthwhile time investment.  It's fairly short, but will take you through the basics of how Terraform works, then into an actual working example.  During the journey, you'll find several key resources which you'll find invaluable as you continue to work with Terraform to manage your OCI environment.
+To explore more information about development with Oracle products:
 
-A picture is worth a thousand words.  Going through this tutorial, you'll be able to better understand why IaC is so cool and has gained so much traction.  You'll also learn how to harness IaC to improve the efficiency of managing your environment, and that alone is worth the time.
+* [Oracle Developers Portal](https://developer.oracle.com/)
+* [Oracle Cloud Infrastructure](https://www.oracle.com/cloud/)
 
-Until our next lesson, happy coding!  Take a look at the [next lesson](2-experiencing-terraform) to go through a very quick experience in using Terraform.  From there we'll dive into several aspects of how Terraform works.
+{% endslides %}
```

Comment:
Oh, are you _adding_ {% endslides %} to posts? This should only exist if the post starts with `{% slides %}` to begin with.

Reply:
No, that was my mistake. I copied and pasted the end section from a previous article that had contained slides.